pacman-nodejs-app/tools/migrate/pacman.sh

```diff
@@ -43,14 +59,41 @@ function set_new_mongo_primary {
         mongo --eval "rs.stepDown(120)" || true
 }
 
+function valid_ip {
```

Comment:
Since this script is sourced in the parent `kmt.sh` script, you don't need to add the function here. The calls below to `valid_ip` will call the function in the parent script.